pennylane/interfaces/torch.py

```diff
@@ -246,3 +258,289 @@ def execute(tapes, device, execute_fn, gradient_fn, gradient_kwargs, _n=1, max_d
         max_diff=max_diff,
     )
     return ExecuteTapes.apply(kwargs, *parameters)
+
+
+def pytreeify(cls):
+    """The Pytree class is used to bypass some PyTorch limitation of `autograd.Function`. The forward pass can only
+    return tuple of tensors but not any other nested structure. This class apply flatten to the forward pass and
+    unflatten the results in the apply function. In this way it is possible to treat multiple tape with multiple
+    measurements.

```

Comment:
```suggestion
    return tuple of tensors but not any other nested structure. This class apply flatten to the forward pass and
    unflatten the results in the apply function. In this way, it is possible to treat multiple tapes with multiple
    measurements.
```
Also
> This class apply flatten to the forward pass and unflatten the results in the apply function.

reads a bit confusing.